.github/workflows/release-process.yml

```diff
@@ -39,11 +39,20 @@ jobs:
           ./hack/build-release.sh
           cd cli
           ./hack/build-binaries.sh
+
+      - name: Run Package build
+        run: |
+          constraintVersion="${{ github.ref_name }}"
+          ./cli/kctrl-linux-amd64 pkg release -y -v ${constraintVersion:1} --debug
+          cd cli
           shasum -a 256 ./kctrl-* | tee -a ../tmp/checksums.txt
           echo "# :open_file_folder: Files Checksum" | tee ../tmp/checksums-formatted.txt
           echo '```' | tee -a ../tmp/checksums-formatted.txt
           cat ../tmp/checksums.txt | tee -a ../tmp/checksums-formatted.txt
           echo '```' | tee -a ../tmp/checksums-formatted.txt
```

Comment:
I think we can keep these in the previous step itself as we are generating the files there, also `cd cli` and `cd ..` won't be needed then.